parlai/core/torch_agent.py

```diff
@@ -239,11 +244,11 @@ class History(object):
         sets the maximum number of tunrs
 
     :param p1_token:
-        token indicating 'person 1'; opt must have 'person_tokens' set to True
+        token indicating 'person 1' (teacher/ others); opt must have 'person_tokens' set to True
         for this to be added
 
-    :param p1_token:
```

Comment:
hmm, was it used to be p1_token for both? Was this a mistake or am I missing something here?